atspi-common/src/interface.rs

```diff
@@ -269,64 +191,6 @@ impl std::ops::BitOr for InterfaceSet {
 	}
 }
 
-impl TryFrom<&str> for Interface {
-	type Error = &'static str;
-
-	fn try_from(s: &str) -> Result<Self, Self::Error> {
-		match s {
-			ACCESSIBLE_INTERFACE_NAME => Ok(Interface::Accessible),
-			ACTION_INTERFACE_NAME => Ok(Interface::Action),
-			APPLICATION_INTERFACE_NAME => Ok(Interface::Application),
-			COLLECTION_INTERFACE_NAME => Ok(Interface::Collection),
-			COMPONENT_INTERFACE_NAME => Ok(Interface::Component),
-			DOCUMENT_INTERFACE_NAME => Ok(Interface::Document),
-			HYPERTEXT_INTERFACE_NAME => Ok(Interface::Hypertext),
-			HYPERLINK_INTERFACE_NAME => Ok(Interface::Hyperlink),
-			IMAGE_INTERFACE_NAME => Ok(Interface::Image),
-			SELECTION_INTERFACE_NAME => Ok(Interface::Selection),
-			SOCKET_INTERFACE_NAME => Ok(Interface::Socket),
-			TABLE_INTERFACE_NAME => Ok(Interface::Table),
-			TABLE_CELL_INTERFACE_NAME => Ok(Interface::TableCell),
-			TEXT_INTERFACE_NAME => Ok(Interface::Text),
-			EDITABLE_TEXT_INTERFACE_NAME => Ok(Interface::EditableText),
-			CACHE_INTERFACE_NAME => Ok(Interface::Cache),
-			VALUE_INTERFACE_NAME => Ok(Interface::Value),
-			REGISTRY_INTERFACE_NAME => Ok(Interface::Registry),
-			DEVICE_EVENT_CONTROLLER_INTERFACE_NAME => Ok(Interface::DeviceEventController),
-			DEVICE_EVENT_LISTENER_INTERFACE_NAME => Ok(Interface::DeviceEventListener),
-			_ => Err("No interface found for conversion."),
-		}
-	}
-}
-
-impl std::fmt::Display for Interface {
```

Comment:
Why have you removed the `Display` implementation? Does `Serialize` auto-implement this?

Reply:
I removed the constants so as to not have duplicate names, so we'd have to write the string literals inside the `Display` impl in order to keep this. But what's the added value of having this? It was only used in tests that I removed, I think the `Debug` impl is enough for users.